src/sdk/models/entityTypes.ts

```diff
@@ -9,6 +24,7 @@ export interface EntityType extends IEntityIdentifier<string>, IAuditedEntity {
 	threshold?: number;
 	readonly?: boolean;
 	disabled?: boolean;
+	patterns?: EntityTypeRegexPattern;
```

Comment:
```suggestion
	patterns?: EntityTypeRegexPattern[];
```

Reply:
Should we split into different types instead like on back-end? e.g. `PatternEntityType`, `TransformEntityType` etc? or are we going to add all properties from all types here?